consul_config.py.ctmpl

```diff
@@ -1,9 +1,9 @@
 {{- define "KEY" -}}
-    {{ key (printf "docker-server-configs/LB/config.%s.json/%s" (env "DEPLOY_ENV") .) }}
+    {{ keyOrDefault (printf "docker-server-configs/LB/config.%s.json/%s" (env "DEPLOY_ENV") .)  "KEYDOESNOTEXIST_{{.}}" }}
```

Comment:
this `{{.}}` syntax isn't correct, it just gets output as a . and not the parameter

Reply:
Looking into it.

Reply:
I think the issue was that this statement is inside another `{{}}`. I have replaced this with a `printf`.